README.md

```diff
@@ -246,7 +246,7 @@ Embedding is done in 2 basic steps: graph construction and training.
 
 Let's assume that the basic configuration of the program looks like this:
     
-    --input files/samples/edgelist_2.tsv --columns users complex::products complex::brands --dimension 3 --number-of-iterations 4
+    --input files/samples/edgelist_2.tsv --columns="users complex::products complex::brands" --dimension 3 --number-of-iterations 4
```

Comment:
Thanks! It can be without `=` as well: `--columns "users complex::products complex::brands"`